pycyphal/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -491,32 +491,21 @@ def _construct_pcan(parameters: _InterfaceParameters) -> can.ThreadSafeBus:
     if isinstance(parameters, _FDInterfaceParameters):
         if parameters.bitrate[0] == 0 or parameters.bitrate[1] == 0:
             raise InvalidMediaConfigurationError("Bitrate must be non-zero")
-        # These magic numbers come from the settings of PCAN adapter.
-        # They don't allow any direct baudrate settings, you have to set all lengths and value of the main frequency.
-        # Bit lengths below are very universal and can be applied for almost every popular baudrate.
-        # There is probably a better solution here, but it needs significantly more time to implement it.
-        f_clock = 40000000
-        nom_tseg1, nom_tseg2, nom_sjw = 3, 1, 1
-        data_tseg1, data_tseg2, data_sjw = 3, 1, 1
-
-        nom_br = int(f_clock / parameters.bitrate[0] / (nom_tseg1 + nom_tseg2 + nom_sjw))
-        data_br = int(f_clock / parameters.bitrate[1] / (data_tseg1 + data_tseg2 + data_sjw))
-        # TODO: validate the result and see if it is within an acceptable range
 
+        timing = can.BitTimingFd.from_sample_point(
+            f_clock=80_000_000,  # TODO: 80 MHz is a good choice for high data rates, what about lower ones?
+            nom_bitrate=parameters.bitrate[0],
+            nom_sample_point=87.5,
```

Comment:
Should we set it to 80% to better conform to CANopen & J1939? Will your setup work in that case?

We should do the same for the data segment I reckon.

Reply:
My setup does not work with 80%, but that is more of a _me_-problem than a pycyphal issue. Setting the default sample point to 80% seems reasonable, as that should be compatible to most setups. I can work with a local fork until my clock issues are resolved...